modules/apps/powwow/powwow-portlet/docroot/WEB-INF/service.xml

```diff
@@ -140,4 +140,30 @@
 
 		<reference entity="User" package-path="com.liferay.portal.kernel" />
 	</entity>
+	<entity local-service="true" name="PowwowMeetingOccurrence" remote-service="true">
+		<!-- PK fields -->
+
+		<column name="occurrenceId" primary="true" type="String" />
+
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
+		<column name="userId" type="long" />
+		<column name="userName" type="String" />
+		<column name="createDate" type="Date" />
+		<column name="modifiedDate" type="Date" />
+
+		<!-- Other fields -->
+
+		<column name="calendarBookingId" type="long" />
+		<column name="powwowMeetingId" type="long" />
+		<column name="zoomOriginalData" type="String" />
+		<column name="occurrenceStatus" type="String" />
+
```

Comment:
for now, i found that we better should have fields for `startTime` and `endTime`, which could be easier for us to handle logic relate to occurrences. Let's add 2 those fields. Please ensure that the time is using the correct time zone, could probably get from `calendarBooking`. In `zoomOriginalData`, i think time zone is default UTC, we may need to convert when storing into occurrence object.